.craft.yml

```diff
@@ -0,0 +1,21 @@
+github:
+  owner: getsentry
+  repo: bundler-plugins
+changelogPolicy: simple
+preReleaseCommand: bash scripts/craft-pre-release.sh
```

Comment:
Technically, this is an optional property as by default, Craft will look for a script called `scripts/bump-version.sh`. However, I think having this written here explicitly as it makes it clear what this script is doing in the repo and where it is needed.